pom.xml

```diff
@@ -50,7 +50,7 @@
       <dependency>
         <groupId>com.datastax.oss</groupId>
         <artifactId>native-protocol</artifactId>
-        <version>1.4.0</version>
+        <version>1.4.1-SNAPSHOT</version>
```

Comment:
This requires a local install of native-protocol. I'll do the release at the last minute in case other changes come up.